src/drivers/intel/cavs/interrupt.c

```diff
@@ -73,26 +81,20 @@ static inline void irq_lvl2_handler(void *data, int level, uint32_t ilxsd,
 
 			if (child && child->handler) {
 				child->handler(child->handler_arg);
-				unmask = child->unmask;
 			} else {
 				/* nobody cared ? */
 				trace_irq_error("irq_lvl2_handler() error: "
-						"nbc");
+						"nobody cared level %d bit %d",
+						level, i);
+				/* now mask it */
```

Comment:
mask it or unmask it here? the comment shows the masking intention, but the code is doing unmasking?

Reply:
It's masking.